### DISCUSSION TITLE
Integration einer Wetter API

### DISCUSSION THREAD

**Discussion by benedictweis** [SE-TINF22B2/G5-DuoGradus#7] [Anforderungen]
Die Anwendung soll, abhängig von dem Wetter, den Nutzer daran Erinnern wie das Wetter ist und was für ein Guter Zeitpunkt es wäre um Aktiv zu werden. Zusätzlich sollte die Anwendung den Nutzer loben falls es bei schlechtem Wetter dennoch viele Schritte gesammelt hat.

**Comment by wwwLuis** [comment]
kann man auch mit den Erfolgen verknüpfen ("Nicht aus Zucker" oder so)
